wowchemy/assets/js/wowchemy-slides.js

```diff
@@ -0,0 +1,146 @@
+import * as params from "@params";
+
+import {fixMermaid} from './wowchemy-utils';
+
+// Enable core slide features.
+enabledPlugins = [
+  RevealMarkdown,
+  RevealHighlight,
+  RevealSearch,
+  RevealNotes,
+  RevealMath,
+  RevealZoom,
+];
+
+const isObject = function (o) {
+  return o === Object(o) && !isArray(o) && typeof o !== "function";
+};
+
+const isArray = function (a) {
+  return Array.isArray(a);
+};
+
+const toCamelCase = (s) => {
+  return s.replace(/([-_][a-z])/gi, ($1) => {
+    return $1.toUpperCase().replace("-", "").replace("_", "");
+  });
+};
+
+const keysToCamelCase = function (o) {
+  if (isObject(o)) {
+    const n = {};
+
+    Object.keys(o).forEach((k) => {
+      n[toCamelCase(k)] = keysToCamelCase(o[k]);
+    });
+
+    return n;
+  } else if (isArray(o)) {
+    return o.map((i) => {
+      return keysToCamelCase(i);
+    });
+  }
+
+  return o;
+};
+
+// reveal configurations can be included in front matter under slides.reveal
+var pluginOptions = {}
+if(typeof params.slides.reveal_options === "undefined"){
+  pluginOptions = params.slides.reveal_options;
```

Comment:
What is the purpose of checking for undefined and then setting the option to undefined? Did you mean to set it to `{}` from `pluginOptions`?

Reply:
Did you mean **not** undefined?

Reply:
yep .. that's what I meant 👍 
Not defined

Reply:
The undefined checks are inconsistently implemented. Sometimes you have used `typeof`, sometimes you haven't. Please update all the undefined checks to be consistent.